app/views/shared/_add_repos_list.html.slim

```diff
@@ -15,7 +15,7 @@
             td= repo["full_name"]
             td #{repo["open_issues_count"]} issues
             td
-              - if Repo.exists_with_name?(repo["full_name"])
+              - if Repo.exists_with_name?(repo["full_name"], current_user.repos)
                 p Added!
```

Comment:
We can't pass current user's repos here, because some other user might have added the same repo already. I think original logic of checking with all repos in the table was correct.